sdk/communication/Azure.Communication.MediaComposition/api/Azure.Communication.MediaComposition.netstandard2.0.cs

```diff
@@ -1,3 +1,26 @@
+namespace Azure.Communication
```

Comment:
The namespace change is a breaking change.
And I suppose we need it to be `Azure.Communication` to be able to work with it in the serializer, right?

Can you pls check with the media composition team, if they've already GA'ed their API or if we can make this change.

adding @DominikMe for visibility. pls consult with him while I'm away

Reply:
looking at their [readme](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/communication/Azure.Communication.MediaComposition), I don't think media composition is even in private preview so I think it should be safe to make this change

Reply:
Yes, it's okay to change this. The MediaComposition SDK hasn't been released yet to anybody.

Reply:
Thank you @DominikMe and @petrsvihlik !